src/lib/holocene/toggle-button/toggle-button.svelte

```diff
@@ -1,99 +1,54 @@
 <script lang="ts">
-  import type {
-    HTMLAnchorAttributes,
-    HTMLButtonAttributes,
-  } from 'svelte/elements';
-
   import { twMerge as merge } from 'tailwind-merge';
 
   import { page } from '$app/stores';
 
-  import type { IconName } from '$lib/holocene/icon';
-  import Icon from '$lib/holocene/icon/icon.svelte';
   import { getAppContext } from '$lib/utilities/get-context';
 
-  import Tooltip from '../tooltip.svelte';
+  import type {
+    ButtonWithHrefProps,
+    ButtonWithoutHrefProps,
+  } from '../button.svelte';
+  import Button from '../button.svelte';
 
   type BaseProps = {
-    icon?: IconName;
     group?: boolean;
     active?: boolean;
-    'data-testid'?: string;
-    tooltip?: string;
-    class?: string;
   };
 
   type AnchorProps = BaseProps &
-    HTMLAnchorAttributes & {
-      href: string;
+    ButtonWithHrefProps & {
       base?: string;
     };
 
   type ButtonProps = BaseProps &
-    HTMLButtonAttributes & {
-      href?: never;
+    ButtonWithoutHrefProps & {
       base?: never;
     };
 
   type $$Props = AnchorProps | ButtonProps;
 
   let className = '';
   export { className as class };
-  export let icon: IconName = null;
   export let group = getAppContext('group');
-  export let href = '';
+  export let href = null;
   export let base = href;
   export let active = false;
-  export let tooltip = '';
+  export let variant: Button['variant'] = 'secondary';
 </script>
 
-<svelte:element
-  this={href ? 'a' : 'button'}
-  class={merge('toggle-button', className)}
-  class:group
-  class:active={href ? $page.url.pathname.includes(base) : active}
-  href={href ? href + $page.url.search : null}
-  class:disabled={$$restProps.disabled}
-  data-track-name="toggle-button"
-  data-track-intent="toggle"
-  data-track-text="*textContent*"
+<Button
   on:click
-  role="button"
-  tabindex="0"
+  class={merge(
+    (href ? $page.url.pathname.includes(base) : active) &&
+      'bg-interactive-secondary-active',
+    group && '[&:not(:last-child)]:border-r-0',
```

Comment:
I think this ternary can be expressed like `href  && ($page.url.pathname.includes(base) || active) && 'css'`